setup/setup.md

```diff
@@ -0,0 +1,41 @@
+# Video Setup for MtlPy
+
+## Driving aspects
+
+- Make amazing online videos: perfect sound, perfect slide capture
```

Comment:
I would add a link to our video (https://www.youtube.com/user/MontrealPython/videos)